README.md

```diff
@@ -1027,64 +1027,163 @@ variables:
       "ummVersion": "1.2.0"
     }
 
+
+#### Associations
+For all supported arguments and columns, see [the schema](src/types/association.graphql).
+
+The association support associate a Tool, Variable or Service record to a collation record. 
```

Comment:
Maybe "The association supports associating a Tool, Variable, or Service record to a collection."